src/pages/Explore_Regions.py

```diff
@@ -119,54 +127,101 @@ def plot_comp(m_route, cause, df):
 
 #  Markdown for the page
 st.header("Explore One Region at a Time")
-st.markdown("In this page we welcome you to explore the data one region at a time, using the filters in the left side menu to filter. Try using the filters and check how the data visualization changes." +
-"Migrations that cross borders and routes include entire families with children. Those routes are so dangerous, and even more for infants." + "On the other hand, others manage to survive the journey, but the fact that they do does not mean that it was without great difficulty and that many others did not.")
+st.markdown("In this page we welcome you to explore the data one region at a time, using the filters in the left side menu. Try selecting a new region to check how the data visualization changes. " +
+            "Migrations that cross borders and routes often include families with children. While extremely dangerous, they are even moreso for infants." + " For those that do survive the journey, it is not without great difficulty and loss.")
+
+st.markdown("The following is a table of all included migration routes paired with the total number of recorded dead and missing people from that route, since 2014.")
+dff = df.groupby(['Migration route'])[
+    ['Total Number of Dead and Missing']].sum().sort_values('Total Number of Dead and Missing', ascending=False).reset_index()
+st.dataframe(dff)
+
 st.sidebar.write("Data Filters")
+
 route_input = [st.sidebar.selectbox(
     'Migration Route', df['Migration route'].unique().tolist())]
+st.header('Exploring the '+route_input[0])
+
+
+if route_input:
+    route_s = route_input[0]
+    mdf = df[df['Migration route'] == route_s]
+    migrantdf = mdf.groupby(['Cause of Death'])[
+        'Total Number of Dead and Missing'].sum().reset_index().sort_values(by='Total Number of Dead and Missing', ascending=False)
+    route_s = route_input[0]
+    worst_month = mdf.groupby(['Migration route', 'date'])['Total Number of Dead and Missing'].sum(
+    ).reset_index().sort_values(by='Total Number of Dead and Missing', ascending=False)
+    df_d_s = mdf.groupby(['Migration route'])[
+        ['Total Number of Dead and Missing', 'Number of Survivors', 'Number of Females', 'Number of Males', 'Number of Children']].sum().reset_index()
+
+    total_dead_missing = prettify(
+        df_d_s['Total Number of Dead and Missing'].iloc[0])
+    total_women_dead_missing = prettify(df_d_s['Number of Females'].iloc[0])
+    total_men_dead_missing = prettify(df_d_s['Number of Males'].iloc[0])
+    total_children_dead_missing = prettify(
+        df_d_s['Number of Children'].iloc[0])
+
+    # introduction
+    st.markdown("For the " + route_input[0] +
+                " migration route, there are a total of  " + total_dead_missing +
+                " recorded deaths in the data since 2014. This includes " + total_women_dead_missing +
+                " women, " + total_men_dead_missing + " men, and " + total_children_dead_missing + " children.")
```

Comment:
Intro to the region includes breakdown of dead/missing by statistics on children/women/men